client/scss/layouts/_login.scss

```diff
@@ -23,32 +33,60 @@ $desktop-nice-padding: 100px;
   }
 
   h1 {
-    font-weight: 300;
-    font-size: 2em;
-    line-height: 1em;
-    color: $color-white;
+    color: theme('colors.primary.DEFAULT');
     white-space: nowrap;
+    font-weight: 800;
+    font-size: 30px;
+    line-height: 130%;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex: none;
+    flex-grow: 0;
+    z-index: 1;
```

Comment:
Why is this needed? This creates a new stacking context and can cause styles to be overly complex.